runtime/bindings/python/tests/test_ngraph/test_manager.py

```diff
@@ -30,8 +32,141 @@ def test_constant_folding():
+
+    os.remove(xml_path)
+    os.remove(bin_path)
+
+
+def test_serialize_pass_mixed_args_kwargs():
+    core = Core()
+    xml_path = "serialized_function.xml"
+    bin_path = "serialized_function.bin"
+    shape = [3, 2]
+    parameter_a = ov.parameter(shape, dtype=np.float32, name="A")
+    parameter_b = ov.parameter(shape, dtype=np.float32, name="B")
+    model = parameter_a - parameter_b
+    func = Function(model, [parameter_a, parameter_b], "Function")
+
+    pass_manager = Manager()
+    pass_manager.register_pass("Serialize", xml_path, bin_path)
+    pass_manager.run_passes(func)
+
+    res_func = core.read_model(model=xml_path, weights=bin_path)
+
+    assert func.get_parameters() == res_func.get_parameters()
+    assert func.get_ordered_ops() == res_func.get_ordered_ops()
+
+    os.remove(xml_path)
+    os.remove(bin_path)
+
+
+def test_serialize_pass_mixed_args_kwargs_v2():
+    core = Core()
+    xml_path = "./serialized_function.xml"
+    bin_path = "./serialized_function.bin"
+    shape = [100, 100, 2]
+    parameter_a = ov.parameter(shape, dtype=np.float32, name="A")
+    parameter_b = ov.parameter(shape, dtype=np.float32, name="B")
+    parameter_c = ov.parameter(shape, dtype=np.float32, name="C")
+    parameter_d = ov.parameter(shape, dtype=np.float32, name="D")
+    model = ov.floor(ov.minimum(ov.abs(parameter_a), ov.multiply(parameter_b, parameter_c)))
+    func = Function(model, [parameter_a, parameter_b, parameter_c], "Function")
+    pass_manager = Manager()
+    pass_manager.register_pass("Serialize", xml_path=xml_path, bin_path=bin_path)
+    pass_manager.run_passes(func)
+
+    res_func = core.read_model(model=xml_path, weights=bin_path)
+
+    assert func.get_parameters() == res_func.get_parameters()
+    assert func.get_ordered_ops() == res_func.get_ordered_ops()
+
+    os.remove(xml_path)
+    os.remove(bin_path)
+
+
+# def test_serialize_pass_wrong_num_of_args():
+    # xml_path = "serialized_function.xml"
+    # bin_path = "serialized_function.bin"
+
+    # pass_manager = Manager()
+    # with pytest.raises(RuntimeError) as e:
+        # pass_manager.register_pass(pass_name="Serialize", xml_path=xml_path, bin_path=bin_path, model=5)
+    # assert "Invoked with too many arguments!" in str(e.value)
+
+
+# def test_serialize_pass_no_paths():
+    # pass_manager = Manager()
+    # with pytest.raises(RuntimeError) as e:
+        # pass_manager.register_pass(pass_name="Serialize")
+    # assert "No required paths specified for saving" in str(e.value)
+
+
+def test_serialize_results():
+    core = Core()
+    node_constant = ov.constant(np.array([[0.0, 0.1, -0.1], [-2.5, 2.5, 3.0]], dtype=np.float32))
+    node_ceil = ov.ceiling(node_constant)
+    func = Function(node_ceil, [], "Function")
+
+    xml_path = "serialized_function.xml"
+    bin_path = "serialized_function.bin"
+    pass_manager = Manager()
+    pass_manager.register_pass("Serialize", xml_path=xml_path, bin_path=bin_path)
```

Comment:
what about the next test: `pass_manager.register_pass("Serialize", output_file=(xml_path, bin_path))`